src/fontMetrics.js

```diff
@@ -198,10 +198,21 @@ const getCharacterMetrics = function(
     let ch = character.charCodeAt(0);
     if (character[0] in extraCharacterMap) {
         ch = extraCharacterMap[character[0]].charCodeAt(0);
-    } else if (cjkRegex.test(character[0])) {
-        ch = 'M'.charCodeAt(0);
     }
-    const metrics = metricMap[font]['' + ch];
+    let metrics = metricMap[font][ch];
+
+    if (!metrics) {
+        // We don't typically have font metrics for Asian scripts.
+        // So if the character is in a script we support but we
+        // dont have metrics for it, just use the metrics for
+        // the Latin capital letter M. This is close enough because
+        // we (currently) only care about the height of the glpyh
+        // not its width.
+        if (supportedCodepoint(ch)) {
+            metrics = metricMap[font][77]; // 77 is the charcode for 'M'
+        }
```

Comment:
I see, we fall back to the metrics for `'M'` for any code point.  Good enough for now.

Reply:
This is really cool and yet I have a question. Many, perhaps most, of these characters are not in Times New Roman, so we don't even know which font will display. An yet at least some of these characters are taller than an "M".
![height1](https://user-images.githubusercontent.com/16403058/34923932-f9754e88-f954-11e7-8cd7-a16e3b11bef9.PNG)

Is there a widely-used system font that contains all these characters?  Then we could specify it in KaTeX CSS and get some font metrics, at least `height` and `depth`.